src/header.tsx

```diff
@@ -173,47 +174,63 @@ export const HeaderLayout: React.FC<HeaderLayoutProps> = ({
-                    <Stack space={24}>
-                        {header}
-                        {extra}
-                    </Stack>
-                </Box>
-            ) : sideBySideExtraOnDesktop ? (
-                <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={48}>
-                    <GridLayout
-                        template="6+6"
-                        left={
+        <div>
+            <ResponsiveLayout
+                isInverse={isInverse}
+                dataAttributes={{'component-name': 'HeaderLayout', ...dataAttributes}}
+            >
+                <OverscrollColor />
+                {isTabletOrSmaller ? (
+                    <Box paddingTop={header ? 32 : 0} paddingBottom={24}>
+                        <Stack space={24}>
+                            {header}
+                            {!bleed && extra}
+                        </Stack>
+                    </Box>
+                ) : sideBySideExtraOnDesktop ? (
+                    <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={48}>
+                        <GridLayout
+                            template="6+6"
+                            left={
+                                <Stack space={32}>
+                                    {breadcrumbs}
+                                    {header}
+                                </Stack>
+                            }
+                            right={extra}
+                        />
+                    </Box>
+                ) : (
+                    <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={bleed ? 32 : 48}>
+                        <Stack space={isTabletOrSmaller ? 24 : 32}>
                             <Stack space={32}>
                                 {breadcrumbs}
                                 {header}
                             </Stack>
-                        }
-                        right={extra}
-                    />
-                </Box>
-            ) : (
-                <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={48}>
-                    <Stack space={isTabletOrSmaller ? 24 : 32}>
-                        <Stack space={32}>
-                            {breadcrumbs}
-                            {header}
+                            {!bleed && extra}
                         </Stack>
-                        {extra}
-                    </Stack>
-                </Box>
+                    </Box>
+                )}
+            </ResponsiveLayout>
+            {bleed && extra && (isTabletOrSmaller || !sideBySideExtraOnDesktop) && (
+                <ResponsiveLayout
+                    backgroundColor={
+                        isInverse
+                            ? `linear-gradient(to bottom, ${vars.colors.backgroundBrand} 40px, ${vars.colors.background} 0%)`
+                            : vars.colors.background
+                    }
+                >
+                    <Box paddingBottom={isTabletOrSmaller ? 24 : 48}>
```

Comment:
@yceballost, it feels weird to have a `paddingBottom` when `bleed` is enabled, what do you think?
![image](https://user-images.githubusercontent.com/1849576/225311228-4a0446ad-e512-4b12-87a0-54d586d23fbf.png)

Reply:
In the bleed specs this padding doesn't exists. I think is better remove this padding to avoid mix spacing with vertical rhythm

Reply:
ok, please @geremiascorrea-tlf, remove this `<Box>`